examples/redux-saga/dynamic-counters/src/App.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import Counter from './components/Counter'
+import { ADD_COUNTER } from './actionTypes'
+
+const initialState = { counterIds: ['Counter one', 'Counter two'] }
+
+export const pageReducer = (state = initialState, action) => {
```

Comment:
move to the `reducers` folder?

Reply:
Also not sure on the `page` name.  `counters`?

Reply:
moved into `reducers/index.js`
un-defaulted previous reducer and renamed to `counterInstanceReducer`
renamed state slice from `page` to `counters`
renamed id array value from `counterIds` to `allIds`